### DISCUSSION TITLE
App crashes

### DISCUSSION THREAD

**Discussion by akshaybabloo** [community/community#8439] [Mobile] [answered]
For some reason the app crashes when I try to open https://github.com/exiftool/exiftool link from Google. And works fine with other GitHub repository links.

I am guessing it's because of the README file.

**Comment by Iron-Ham** [comment]
@akshaybabloo are you experiencing this on iOS or Android? Could you share which version of the application you're currently using?

**Reply by akshaybabloo** [reply]
This is on iOS. On the application version 1.43.0

**Reply by Iron-Ham** [reply, marked as answer]
I've identified the crash, and have merged a fix for it. Keep an eye out for it in the next release!

**Reply by akshaybabloo** [reply]
Awesome. Thanks.
